content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -160,7 +160,7 @@ To create a new Secret, perform the following steps:
    You will need to mount the new encryption config file to the `kube-apiserver` static pod. Here is an example on how to do that:
 
    1. Save the new encryption config file to `/etc/kubernetes/enc/enc.yaml` on the control-plane node.
-   1. Edit the manifest for the `kube-apiserver` static pod: `/etc/kubernetes/manifests/kube-apiserver.yaml` similarly to this:
+   1. Edit the manifest for the `kube-apiserver` static pod. `/etc/kubernetes/manifests/kube-apiserver.yaml` is similar to this:
```

Comment:
The existing text seems better IMO.

Reply:
ok, reverted though it's a little weird to me.